docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/docstring_flags.py

```diff
@@ -100,12 +106,9 @@ class flag(nodes.Element):
 
 def visit_flag(self, node: flag):
     flag_type, message = [node.attributes[k] for k in FLAG_ATTRS]
-    # We are currently not parsing the content of the message, so manually sub
-    # all `references` with `<cite>` tags, which is what the HTML writer does
-    # for parsed RST.
-    message = re.sub(r"`(\S+?)`", r"<cite>\1</cite>", message)
```

Comment:
Not sure how relevant this is, but to format on new lines, I removed this preprocessing step.